python/beeai_framework/adapters/litellm/chat.py

```diff
@@ -226,7 +227,7 @@ def _transform_output(self, chunk: ModelResponse | ModelResponseStream) -> ChatM
                 else []
             ),
             finish_reason=finish_reason,
-            usage=usage,
+            usage=ChatModelUsage(**usage.model_dump()) if usage else None,
```

Comment:
For reference: `usage` is a `litellm.types.utils.Usage` object, which extends BaseModel and contains all the expected keys in `ChatModelUsage`. So this line copies those values into a new `ChatModelUsage` object. This would also allow us to update `ChatModelUsage` at anytime to include more values in `Usage`

Reply:
Great. But if they suddenly change the structure, it will cause a breaking change for us.

Reply:
That would still be the case if we manually look up each value in the class though, unless they update it to not extend BaseModel, but they leverage BaseModel features on that class in their own implementation so that is unlikely